audit-ci.json

```diff
@@ -1,6 +1,5 @@
 {
   "low": true,
   "package-manager": "auto",
-  "registry": "https://registry.npmjs.org",
-  "allowlist": ["GHSA-pfrx-2q88-qq97|nodemon>update-notifier>latest-version>package-json>got"]
```

Comment:
according to https://github.com/ministryofjustice/hmpps-template-typescript/pull/124/files no longer relevant